docs/designers-developers/developers/tutorials/create-block/block-anatomy.md

```diff
@@ -1,6 +1,10 @@
-# Anatomy of a Gutenberg Block
+# Anatomy of a Block
 
-At its simplest, a block in Gutenberg is a JavaScript object with a specific set of properties. Here is the complete code for registering a block:
+At its simplest, a block in Gutenberg is a JavaScript object with a specific set of properties.
```

Comment:
Do we still try to replace Gutenberg with the references to the block editor in the official docs? It would be then:
```suggestion
At its simplest, a block in the WordPress block editor is a JavaScript object with a specific set of properties.
```